src/Compilers/CSharp/Portable/Symbols/TypeSymbol.cs

```diff
@@ -1090,6 +1090,24 @@ private static void ReportImplicitImplementationMatchDiagnostics(Symbol interfac
                         ReportAnyMismatchedConstraints(interfaceMethod, implementingType, implicitImplMethod, diagnostics);
                     }
                 }
+
+                if (implicitImpl.ContainingType.IsInterface && implementingType.ContainingModule != implicitImpl.ContainingModule)
+                {
+                    // PROTOTYPE(DefaultInterfaceImplementation): Should we check language version as well?
+                    // Usually, it is done based on specific syntax that targets a new feature, but in this case
+                    // no special syntax is used. Also, partial types can have declarations coming from different 
```

Comment:
Yes, I agree that there is no special syntax at this site. But there is a new language feature being used. In any case I agree with your decision to document it as an open issue with a PROTOTYPE comment.